src/Services/Forms/BladePartial.php

```diff
@@ -30,13 +32,19 @@ public function render(): ?View
             return null;
         }
 
-        $form = ViewFacade::shared("form");
-
-        return ViewFacade::make($this->view, [
-            'item' => $form['item'] ?? null,
-            'form_fields' => $form['form_fields'] ?? [],
-            'formModuleName' => $form['moduleName'] ?? null,
-            'routePrefix' => $form['routePrefix'] ?? null,
-        ]);
+        $form = ViewFacade::shared('form');
+
+        return ViewFacade::make(
+            $this->view,
+            array_merge(
```

Comment:
Thanks @sauron this is a great and needed addition. I wonder if it would be better to reverse the parameters in this merge, so that, if one needs that, they can override the parameters that Twill is passing by default.

Reply:
Makes total sense!
Done!